gunrock/app/hits/hits_test.cuh

```diff
@@ -47,14 +51,20 @@ double CPU_Reference(const GraphT &graph, typename GraphT::ValueT *ref_hrank,
   ValueT *next_hrank = new ValueT[graph.nodes];
   ValueT *next_arank = new ValueT[graph.nodes];
 
-  // Set next scores to 1 and 0
+  // Set initial scores
+  for (SizeT v = 0; v < graph.nodes; v++) {
+    curr_hrank[v] = 1.0/graph.nodes;
+    curr_arank[v] = 1.0/graph.nodes;
+  }
+
   for (SizeT v = 0; v < graph.nodes; v++) {
-    curr_hrank[v] = 1.0;
-    curr_arank[v] = 1.0;
+    next_hrank[v] = 0.0;
+    next_arank[v] = 0.0;
   }
 
-  for (SizeT iterCount = 0; iterCount < max_iter; iterCount++) {
-    // Set next scores to 1 and 0
+  SizeT iterCount = 0;
+  for (iterCount = 0; iterCount < max_iter; iterCount++) {
```

Comment:
Put the initialization into the for-loop like usual.

Reply:
Moved the initialization into the for-loop, but declaration needs to be outside the loop since later sections of the code need to know the final value of this variable.

Reply:
Ah gotcha. Now I see.